packages/@react-stately/radio/src/useRadioGroupState.ts

```diff
@@ -76,6 +82,7 @@ export function useRadioGroupState(props: RadioGroupProps): RadioGroupState  {
     isDisabled: props.isDisabled || false,
     isReadOnly: props.isReadOnly || false,
     isRequired: props.isRequired || false,
-    validationState: props.validationState || null
+    validationState: props.validationState || null,
+    isInvalid: props.isInvalid || props.validationState === 'invalid'
```

Comment:
these should probably all be like this?
```suggestion
    isInvalid: props.isInvalid ?? props.validationState === 'invalid'
```

Reply:
hmm not sure we should treat `false` and `undefined` differently. I think it's ok if it's invalid if either prop says so.